docs/product-user-guide/src/main/asciidoc/business-central-home-con.adoc

```diff
@@ -2,61 +2,162 @@
 = Business Central Home
 
 ifdef::BPMS[]
-The {PRODUCT} *Home* view consists of a main menu with various drop-down options or "perspectives" that you use to author and deploy business process projects, create and manage tasks and dashboards, and integrate plugins.
+The {PRODUCT} Home view provides links to various options, called perspectives, that you use to author and deploy business process projects, create and manage tasks and dashboards, and integrate plugins.
 
 .{PRODUCT} Home
-image::6592.png[BPM Home Screen]
+image::user-guide-homepage.png[BPM Home Screen]
 endif::BPMS[]
 
 ifdef::BRMS[]
-The {PRODUCT} *Home* view consists of a main menu with various drop-down options or "perspectives" that you use to author and deploy business rule projects, create and manage tasks, and integrate plugins.
+The {PRODUCT} Home view provides links to various options, called perspectives, that you use to author and deploy business rule projects, create and manage tasks, and integrate plugins.
 
 .{PRODUCT} Home
-image::3271.png[BRMS Home Screen]
+image::user-guide-homepage.png[BRMS Home Screen]
 endif::BRMS[]
 
-<1> The main menu contains the links to the `Home` page and all available perspectives.
-<2> The perspective menu contains menus for the selected perspective.
-<3> The perspective area contains the perspective tools, such as views and editors.
+In addition, click *Menu* in the top left of the screen to display a list of all of the perspectives. 
```

Comment:
Shouldn't it be "on the top left of the screen"?

Reply:
I'd says it's 'in'. IBMSG doesn't directly specify but uses 'in' by example. (In the window...) In any case both are used in the doc so we need to decided later and make a global change. Leaving for now.